etc/logrotate.d/icinga2.cmake

```diff
@@ -6,7 +6,7 @@
 	missingok
 	notifempty@LOGROTATE_CREATE@
 	postrotate
-		/bin/kill -USR1 $(cat @ICINGA2_INITRUNDIR@/icinga2.pid 2> /dev/null) 2> /dev/null || true
+		@CMAKE_INSTALL_FULL_SBINDIR@/icinga2 internal signal --sig SIGUSR1 --pid "$(cat @ICINGA2_INITRUNDIR@/icinga2.pid 2> /dev/null)" 2> /dev/null || true
```

Comment:
Be aware that this is a config file which we've currently told the package manager not to override by %config(noreplace).

Reply:
If the file was modified, there was probably a reason for doing so. Just overwriting it would likely break something.

Reply:
And just keeping it leaves a security hole(?) for all who don't read the changelog, but modified stuff. Also, %config [makes a backup](https://www.cl.cam.ac.uk/~jw35/docs/rpm_config.html) of your modifications.